addons/spreadsheet_dashboard/tests/test_spreadsheet_dashboard.py

```diff
@@ -21,6 +21,21 @@ def test_create_with_default_values(self):
             b'{"version": 1, "sheets": [{"id": "sheet1", "name": "Sheet1"}]}',
         )
 
+    def test_write_raw_data(self):
+        group = self.env["spreadsheet.dashboard.group"].create(
+            {"name": "a group"}
+        )
+        dashboard = self.env["spreadsheet.dashboard"].create(
+            {
+                "name": "a dashboard",
+                "dashboard_group_id": group.id,
+            }
+        )
+        data = b'{"version": 1, "sheets": [{"id": "sheet1", "name": "Sheet1"}]}'
+        dashboard.raw = data
+        self.assertEqual(dashboard.raw, data)
```

Comment:
Maybe a comment that we are testing computed field magic (that's what we do right ?) ? Because at first glance it looks like the assert is pointless 😅